qt/scientific_interfaces/Indirect/IndirectTab.h

```diff
@@ -88,16 +88,6 @@ class MANTIDQT_INDIRECT_DLL IndirectTab : public QObject, public IPyRunner {
 
   void displayWarning(std::string const &message);
 
-public slots:
```

Comment:
Should this have gone in?

Reply:
I don't know why this is showing as being removed. It should still be there. If you look at `IndirectTab` in the Files Changed it seems to be fine

Reply:
Strange, perhaps some mix up during the rebase made github think it ahd been changed. it is fine now.